plugins/node/instrumentation-cucumber/package.json

```
@@ -59,7 +59,7 @@
     "mocha": "7.2.0",
     "nyc": "15.1.0",
     "rimraf": "5.0.1",
-    "semver": "^7.3.2",
+    "semver": "^7.5.4",
```

Since this is a dev dependency, it should be pinned:
```suggestion
    "semver": "7.5.4",
```